notebooks/406-3D-pose-estimation-webcam/406-opencv-3D-pose-estimation.ipynb

```diff
@@ -0,0 +1,572 @@
+    "            player.stop()\n",
+    "        if use_popup:\n",
+    "            cv2.destroyAllWindows()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a2f9f89f-c65e-4625-abd5-93b442b27e1b",
+   "metadata": {},
+   "source": [
+    "## Run\n",
+    "\n",
+    "### Run Live Pose Estimation\n",
+    "\n",
+    "Run using a webcam as the video input. By default, the primary webcam is set with `source=0`. If you have multiple webcams, each one will be assigned a consecutive number starting at 0. Set `flip=True` when using a front-facing camera. Some web browsers, especially Mozilla Firefox, may cause flickering. If you experience flickering, set `use_popup=True`.\n",
+    "\n",
+    "*Note: If you set `use_popup=True`, use your keyboard to move your camera, like:\n",
+    "\n",
+    "\n",
+    "<table >\n",
+    "    <tr>\n",
+    "        <th> key <th><th> function <th><th> key <th><th> function <th>\n",
+    "    <tr>\n",
+    "    <tr> \n",
+    "        <td>'W'<td><td>forward<td><td>'I'<td><td>upward <td>\n",
+    "    <tr>\n",
+    "    <tr> \n",
+    "        <td>'S'<td><td>back<td><td>'J'<td><td>towards the left <td>\n",
+    "    <tr>\n",
+    "    <tr> \n",
+    "        <td>'A'<td><td>left<td><td>'K'<td><td>downward <td>\n",
+    "    <tr>\n",
+    "    <tr> \n",
+    "        <td>'D'<td><td>right<td><td>'L'<td><td>towards the right <td>\n",
+    "    <tr>\n",
+    "    <tr> \n",
+    "        <td>'Q'<td><td>up<td><td>'E'<td><td>down <td>\n",
+    "    <tr>\n",
+    "       <table>\n",
+    "<!-- <tr><td> Stretched <td><td> Gaussian <td><td> .843 <td><tr> -->\n",
+    "\n",
+    "\n",
+    "*Note: To use this notebook with a webcam, you need to run the notebook on a computer with a webcam. If you run the notebook on a server (e.g. Binder), the webcam will not work.*\n",
+    "\n",
+    "*Note: Popup mode may not work if you run this notebook on a remote computer (e.g. Binder).*"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3f82e298-5912-48c7-90b5-339aea3c177d",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "run_pose_estimation(source=0, flip=True, use_popup=False, boxheight=300, boxwidth=400)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8365628d-f0a1-4b0c-b3bb-0f16a7208874",
+   "metadata": {},
+   "source": [
+    "### Run Pose Estimation on a Video File\n",
+    "\n",
+    "If you don't have a webcam, you can still run this demo with a video file. Any [format supported by OpenCV](https://docs.opencv.org/4.5.1/dd/d43/tutorial_py_video_display.html) will work. \n",
+    "You can click and move your mouse over the picture on the left to interact."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4f3fddfc-7a23-4897-b57f-81a1ac9aad5e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run_pose_estimation(source=video_path, flip=True, use_popup=False, boxheight=300, boxwidth=550)"
```

Comment:
It should run in a popup every time. Otherwise, the control keys don't work. The webcam should be also in a popup if possible.